docs/core/proto-docs.md

```diff
@@ -1,5 +1,7 @@
 <!-- This file is auto-generated. Please do not modify it yourself. -->
+
```

Comment:
This looks like some good clean up, but it doesn't really belong in this PR (it's bigger than the actual code), let's split off a cleanup/whitespace PR. There are some random typos I would add as well.

Reply:
Current master does not have proto-docs, they moved proto docs into https://buf.build/cosmos/cosmos-sdk/docs/main